ui/src/components/Header/index.tsx

```diff
@@ -1,78 +1,81 @@
-import { Link } from "react-router-dom"
-import MultiProxySelection from "../MultiProxySelection"
-import { useAccounts } from "../../contexts/AccountsContext"
-import styled from "styled-components";
+import {Box, Button, IconButton, Toolbar, Typography} from "@mui/material"
+import styled from "styled-components"
+import * as React from "react";
+import MuiAppBar from '@mui/material/AppBar';
+import MultiProxySelection from "../MultiProxySelection";
+import {useAccounts} from "../../contexts/AccountsContext";
+import {Link} from "react-router-dom";
+import MenuIcon from "@mui/icons-material/Menu";
+import {ROUTES} from "../../constants";
 // import NetworkSelection from "../NetworkSelection"
 
 interface Props {
-  className?: string
+    className?: string
+    handleDrawerOpen: () => void,
 }
 
-const Header = ({ className }: Props) => {
-  const { accountList } = useAccounts()
+const Header = ({className, handleDrawerOpen}: Props) => {
+  const {accountList} = useAccounts()
 
   return (
-    <AppBar className={className}>
-      <Container maxWidth="xl">
-        <Toolbar disableGutters>
-          <Typography
-            variant="h6"
-            noWrap
-            component="div"
-            sx={{ flexGrow: 1 }}
-          >
+    <MuiAppBar position="fixed" className={className}>
+      <Toolbar>
+        <TypographyStyled
+          variant="h6"
+          noWrap
+        >
             Multix
-          </Typography>
-          {!!accountList?.length && (
-            <>
-              <Box className="buttonContainer">
-                <Button
-                  component={Link}
-                  to="/"
-                  className="buttonHeader"
-                >
-                  Home
-                </Button>
-                <Button
-                  component={Link}
-                  to="/create"
-                  className="buttonHeader"
-                >
-                  New Multisig
-                </Button>
-                <Button
-                  component={Link}
-                  to="/about"
-                  className="buttonHeader"
-                >
-                  About
-                </Button>
-                <Button
-                  component={Link}
-                  to="/help"
-                  className="buttonHeader"
-                >
-                  Help
-                </Button>
-              </Box>
-              <MultiProxySelection />
-            </>
-          )}
-          {/* <NetworkSelection /> */}
-        </Toolbar>
-      </Container>
-    </AppBar>
+        </TypographyStyled>
+        {!!accountList?.length && (
+          <BoxStyled>
+            {ROUTES.map(({url, name}) => (
+              <Button
+                component={Link}
+                to={url}
+                className="buttonHeader"
+              >
+                {name}
+              </Button>
+            ))}
+          </BoxStyled>
+        )}
+        <MultiProxySelection/>
```

Comment:
can we add the `MultiProxySelection` and the `NetworkSelection` to the mobile menu?
The Kusama deployment is imminent and while the mobile view is better without the menu, I think it won't be manageable with both the network selector and the account selector in the top bar.

Reply:
I can do it for sure, but as a design update are pretty close we probably could wait until the final version

Reply:
I think that the design update will really take a while to land, and start, so I think it's a low effort high reward thing to do.